css-typed-om/Overview.bs

```diff
@@ -990,6 +1008,24 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |result|.
 </div>
 
+<div algorithm="CSSNumericValue.type()">
+    The <dfn method for=CSSNumericValue>type()</dfn> method
+    returns a representation of the [=type=] of |this|.
+
+    When called, it must perform the following steps:
+
+    1. Let |result| be a new {{CSSNumericType}} with all attributes set to `null`.
+
+    2. For each |baseType| → |power| in the [=type=] of |this|,
+        1. If |power| is not 0,
+            set |result|'s |baseType| attribute to |power|.
+
+    3. If the [=percent hint=] of |this| is not null,
+        1. Set {{CSSNumericType/percentHint}} to the [=percent hint=] of |this|.
```

Comment:
This is a bit handwavy as it doesn't quite type check - `CSSNumericType.percentHint` is an enum whereas the `percent hint` of `this` is a string (I think).

Reply:
I think that's fine - enums turn into strings when they go to ECMAScript anyway.